### PR TITLE
Make blockwishlist compatible

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Blockwishlist is not fully compatible with bootstrap 5
| Type?             | bug fix
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | Test blockwishlist inside the theme
| Possible impacts? | Blockwishlist

### CODE REVIEW

File: modules/blockwishlist/views/templates/components/modals/add-to-wishlist.tpl [hunk 1]
```diff
@@ -0,0 +1,77 @@
+{**
+ * 2007-2020 PrestaShop and Contributors
```
Comment (matks) [inline modules/blockwishlist/views/templates/components/modals/add-to-wishlist.tpl:2]: 2023 ? 😄 let's anticipate

File: modules/blockwishlist/views/templates/components/modals/create.tpl [hunk 2]
```diff
@@ -0,0 +1,93 @@
+{**
+ * 2007-2020 PrestaShop and Contributors
```
Comment (matks) [inline modules/blockwishlist/views/templates/components/modals/create.tpl:2]: Same

**Review by matks — COMMENTED** [review]
@NeOMakinG Code is OK but license header can be updated

**Review by nicosomb — DISMISSED** [review]

**Comment by NeOMakinG** [comment]
> @NeOMakinG Code is OK but license header can be updated

Damn, I took those from blockwishlist directly, I'll update them as an override is now the responsibility of the theme!

**Review by matthieu-rolland — APPROVED** [review]

**Review by nicosomb — APPROVED** [review]

File: modules/blockwishlist/views/templates/components/modals/rename.tpl [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+           data-dismiss="modal"
+           aria-label="Close"
+         >
+         </button>
+       </div>
+       <div class="modal-body">
+         <div class="form-group form-group-lg">
+           <label class="form-label" for="input2"
+             >((label))</label
+           >
```
Comment (kpodemski) [inline modules/blockwishlist/views/templates/components/modals/rename.tpl:41]:
```suggestion
           <label class="form-label" for="input2">((label))</label>
```

Comment (JBLach) [inline modules/blockwishlist/views/templates/components/modals/rename.tpl:41]:
```suggestion
           <label class="form-label" for="input2">((label))</label>
```

**Review by nicosomb — APPROVED** [review]

**Review by SharakPL — DISMISSED** [review]

**Review by nicosomb — DISMISSED** [review]

File: modules/blockwishlist/views/templates/components/modals/create.tpl [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+ <div
+ class="wishlist-create"
+ data-url="{$url}"
+ data-title="{l s='Create wishlist' d='Modules.Blockwishlist.Shop'}"
+ data-label="{l s='Wishlist name' d='Modules.Blockwishlist.Shop'}"
+ data-placeholder="{l s='Add name' d='Modules.Blockwishlist.Shop'}"
+ data-cancel-text="{l s='Cancel' d='Modules.Blockwishlist.Shop'}"
+ data-create-text="{l s='Create wishlist' d='Modules.Blockwishlist.Shop'}"
+ data-length-text="{l s='List title is too short' d='Modules.Blockwishlist.Shop'}"
+>
```
Comment (SharakPL) [inline modules/blockwishlist/views/templates/components/modals/create.tpl:15]:
```suggestion
<div
  class="wishlist-create"
  data-url="{$url}"
  data-title="{l s='Create wishlist' d='Modules.Blockwishlist.Shop'}"
  data-label="{l s='Wishlist name' d='Modules.Blockwishlist.Shop'}"
  data-placeholder="{l s='Add name' d='Modules.Blockwishlist.Shop'}"
  data-cancel-text="{l s='Cancel' d='Modules.Blockwishlist.Shop'}"
  data-create-text="{l s='Create wishlist' d='Modules.Blockwishlist.Shop'}"
  data-length-text="{l s='List title is too short' d='Modules.Blockwishlist.Shop'}"
>
```

File: modules/blockwishlist/views/templates/components/modals/add-to-wishlist.tpl [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+    tabindex="-1"
+    role="dialog"
+    aria-modal="true"
+  >
+    <div class="modal-dialog modal-dialog-centered" role="document">
+      <div class="modal-content">
+        <div class="modal-header px-3">
+          <h5 class="modal-title">
+            {l s='Add to wishlist' d='Modules.Blockwishlist.Shop'}
+          </h5>
```
Comment (SharakPL) [inline modules/blockwishlist/views/templates/components/modals/add-to-wishlist.tpl:24]:
```suggestion
          <h5 class="modal-title">{l s='Add to wishlist' d='Modules.Blockwishlist.Shop'}</h5>
```

File: modules/blockwishlist/views/templates/components/modals/create.tpl [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+        <div class="modal-body">
+          <div class="form-group form-group-lg">
+            <label class="form-label" for="input2">((label))</label>
+            <input
+              type="text"
+              class="form-control form-control-lg"
+              v-model="value"
+              id="input2"
+              :placeholder="placeholder"
+            />
```
Comment (SharakPL) [inline modules/blockwishlist/views/templates/components/modals/create.tpl:47]:
```suggestion
            <input
              type="text"
              class="form-control form-control-lg"
              v-model="value"
              id="input2"
              :placeholder="placeholder"
            >
```

File: modules/blockwishlist/views/templates/components/modals/login.tpl [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+    </div>
+  </div>
+
+  <div
+    class="modal-backdrop fade"
+    {literal}
+      :class="{in: !isHidden}"
+    {/literal}
+  >
+ </div>
```
Comment (SharakPL) [inline modules/blockwishlist/views/templates/components/modals/login.tpl:63]:
```suggestion
  <div
    class="modal-backdrop fade"
    {literal}
      :class="{in: !isHidden}"
    {/literal}
  >
  </div>
```

File: modules/blockwishlist/views/templates/components/modals/rename.tpl [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+        </div>
+        <div class="modal-body">
+          <div class="form-group form-group-lg">
+            <label class="form-label" for="input2">((label))</label>
+            <input
+              type="text"
+              class="form-control form-control-lg"
+              v-model="value"
+              id="input2"
+            />
```
Comment (SharakPL) [inline modules/blockwishlist/views/templates/components/modals/rename.tpl:45]:
```suggestion
            <input
              type="text"
              class="form-control form-control-lg"
              v-model="value"
              id="input2"
            >
```

File: modules/blockwishlist/views/templates/components/modals/share.tpl [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+        <div class="modal-body">
+          <div class="form-group form-group-lg">
+            <label class="form-label" for="input2">((label))</label>
+            <input
+              type="text"
+              class="form-control form-control-lg"
+              v-model="value"
+              id="input2"
+              placeholder="Share link"
+            />
```
Comment (SharakPL) [inline modules/blockwishlist/views/templates/components/modals/share.tpl:46]:
```suggestion
            <input
              type="text"
              class="form-control form-control-lg"
              v-model="value"
              id="input2"
              placeholder="Share link"
            >
```

**Review by SharakPL — CHANGES_REQUESTED** [review]
Last small fixes

**Comment by NeOMakinG** [comment]
I wish we had a linter for this

**Review by SharakPL — APPROVED** [review]

**Comment by SharakPL** [comment]
> I wish we had a linter for this

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
🍬 @NeOMakinG :D

**Comment by NeOMakinG** [comment]
> 🍬 @NeOMakinG :D

<iframe src="https://giphy.com/embed/3orif180X6PAeh02d2" width="480" height="360" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/season-12-the-simpsons-12x11-3orif180X6PAeh02d2">via GIPHY</a></p>

**Pull request merged**